### ISSUE TITLE
ui5-input: output of screen reader is not expected when number input is focused

### ISSUE THREAD

**Issue by zhangliangyi** [SAP/ui5-webcomponents#3585]
### **Bug Description**
When I use NVDA screen reader to check the output of number field, the speech output is not expected when the number input is focused.

Consider there is a ui5 webcomponent
```html
<ui5-input type="Number" value="10"></ui5-input>
```
When it is focused, the output of the screen reader is:
```
spin button editable selected
```

### **Expected Behavior**
The value of the input should be output via screen reader

### **Steps to Reproduce**
1. Go to the example link below
2. Focus on the number input via keyboard
3. Check the output of screen reader

### **Isolated Example**
https://codesandbox.io/s/ui5-webcomponents-forked-hykx8

### **Context**
- UI5 Web Components version: 0.31.13
- OS/Platform: Window/Mac OS
- Browser: Edge
- Affected component: ui5-input

### **Priority**
- [ ] Low
- [ ] Medium
- [x] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SF
- Bussiness impact: {...}

**Comment by elenastoyanovaa** [comment]
Hello,

The issue is related to NVDA screen reader, not with the web component itself. It is also reproducible with native input as well. What is more, it is reproducible in Chrome also. You can check https://codesandbox.io/s/ui5-webcomponents-forked-htrd3?file=/index.html

The issue is not reproducible with JAWS. There is also a filed issue https://github.com/nvaccess/nvda/issues/5422 and as per the last response, the issue is reproducible still.

Kind Regards,
Elena
